syncdiff

```diff
@@ -61,6 +61,20 @@ sub display_help {
 	print "If you are running this you had better be John 'Warthog9' Hawley\n";
 }
 
+sub debug_mode {
+	require File::Find::Rule;
```

Comment:
I think instead of running through the files and modifying the variable, you may want to set this up as a config file style option.  If it's defined in the config file debug = <statement that is true> OR --debug is set, specifically set something in $config to indicate that debugging output should be turned on.  Since $config gets passed around that should work out fine, and should be more robust than your find and mapping you are doing here.